### DISCUSSION TITLE
Bring AUR PKGBUILDs under one repository

### DISCUSSION THREAD

**Discussion by Nyameliaaaa** [catppuccin/catppuccin#2295] [Ideas]
# AUR
Merge all the AUR port PKGBUILDs under one repository, this discussion is to allow a birds-eye view of the state of it.

## Which AUR Packages do we have?
These packages should be transferred to the `catppuccin` AUR account.

| Port Name       | AUR Package Names                      | Maintainer  | Needs rename? |
|:---------------:|----------------------------------------|-------------|:-------------:|
| ~~GTK~~             | `catppuccin-gtk-theme-{FLAVOR}`        | @ghostx31   |       N       |
| ~~Cursors~~         | `catppuccin-cursors-{FLAVOR}`          | @ghostx31   |       N       |
| ~~Grub [4]~~        | `catppuccin-{FLAVOR}-grub-theme-git`   | @night-lake |       Y       |
| ~~Papirus Folders~~ | `papirus-folders-catppuccin-git`       | @latipun7   |    see [3]    |
| ~~Kvantum~~         | `kvantum-theme-catppuccin-git`         | @latipun7   |     see [1]     |

## Which themes need to be claimed?
| Port Name | AUR Package Names                        | Needs rename? |
|:---------:|------------------------------------------|:-------------:|
| KDE       | `plasma5-theme-catppuccin(-git)`         | N, Arch Rules Override |
| ~~Plymouth~~  | `plymouth-theme-catppuccin-{FLAVOR}-git` | Y             |
| ~~Konsole~~   | `catppuccin-konsole-theme-git`           | see [1]       |
| SDDM      | `sddm-theme-catppuccin-git`              | see [1], [2]  |
| rEFInd    | `refind-theme-catppuccin-git`            | see [1]       |
| Whiskers  | `whiskers-bin`                           | see [3]       |
| Fcitx5    | `catppuccin-fcitx5-git`                  | see [1]       |

## Which ports need to be claimed/adopted and marked OOD?
These ports are superseded by a copy in the port's upstream package manager/plugin manager. None have been marked OOD so far, would probably be in the hands of `core` to write the OOD message.

### Claimed by a member of the Catppuccin Discord
- **Fish** `fish-catppuccin-{FLAVOR}-git`
- **nvim** 
   - `neovim-catppuccin`
   - `neovim-catppuccin-git`
- **Python** `python-catppuccin`
- **Duplicate Cursor Entry** 
   - `catppuccin-mocha-light-cursors`
   - `catppuccin-mocha-dark-cursors` 

### Independently maintained
- **Jupyterlab** `jupyterlab-catppuccin`

> Note:
> If any of these tasks are completed, please update it's state in this document accordingly.

[1]: Not sure if these just install the whole thing/a script, if so, drop `-{FLAVOR}` and maybe indicate as such?
[2]: Get [sddm-catppuccin-git](https://aur.archlinux.org/packages/sddm-catppuccin-git) renamed to `sddm-corners-theme-catppuccin-git` or similar to prevent confusion.
[3]: Do CLI based ports (Whiskers, Papirus Folders) need to be renamed to something like `catppuccin-{TOOL}-bin` for consistency?
~~[4]: Currently, it installs it to `/usr/share/grub/themes/catppuccin-{FLAVOR}` and not `/usr/share/grub/themes/catppuccin-{FLAVOR}-grub-theme` like it is from a repo clone, not sure which direction we should go in~~
5]: All packages are to be renamed with the “-git” suffix if they clone, which most likely do

**Comment by sgoudham** [comment]
Worth mentioning that @night-lake has moved away from arch-linux and I don't believe they're interested anymore in spearheading the monorepo (which is totally fine!)

The repository has been created at https://github.com/catppuccin-rfc/aur-packages but still needs to be setup. 

I believe @ghostx31 is wanting to set up the repository when he gets some free time, along with the on-going effort to claim packages.

**Reply by Nyameliaaaa** [reply]
i am back on arch actually :P

**Comment by sgoudham** [comment]
Just an update on this, [catppuccin/aur-packages](https://github.com/catppuccin/aur-packages) now exists!

We've got a workflow set up using [Renovate](https://www.mend.io/renovate/) to automatically pull in updates from our repositories and automatically publish to the AUR! For more information on what we based it on, see "[Maintaining AUR Packages with Renovate](https://docs.renovatebot.com/user-stories/maintaining-aur-packages-with-renovate/)".

We're now looking to start adding packages aside from our GTK theme so we can automatically publish them too! Any help is appreciated. I'll be writing up some documentation on how the repository works and how to easily contribute!
